index.html

```diff
@@ -5,271 +5,374 @@
+              </svg>
+            </div>
+            <div>
+              <ul class="langs">
+                <li>
+                  <div class="lang">
+                    <img alt="js" src="./assets/js.png">
+                    <p>JavaScript</p>
+                  </div>
+                </li>
+                <li>
+                  <div class="lang">
+                    <img alt="html" src="./assets/html.png">
+                    <p>HTML</p>
+                  </div>
+                <li>
+                  <div class="lang">
+                    <img alt="css" src="./assets/css.png">
+                    <p>CSS</p>
+                  </div>
+                </li>
+              </ul>
+            </div>
+          </li>
+          <li class="list">
+            <div class="dropdown-menu border-b">
+              <p class="lang-text">Frameworks</p>
+              <svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 20 20" fill="currentColor">
+                <path fill-rule="evenodd"
+                  d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
+                  clip-rule="evenodd" />
+              </svg>
+            </div>
+          </li>
+          <li class="list">
+            <div class="dropdown-menu border-b">
+              <p class="lang-text">Skills</p>
+              <svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 20 20" fill="currentColor">
+                <path fill-rule="evenodd"
+                  d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
+                  clip-rule="evenodd" />
+              </svg>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <footer class="contact-section-bg">
+      <div class="contact-section">
+        <div class="contact">
+          <h2 class="title">Contact me</h2>
+          <p class="details">If you have an application you are interested in developing, a feature that you need
+            built or a
+            project that needs coding. I’d love to help with it </p>
+          <form action="https://formspree.io/f/mjvjzbal" method="POST" class="form">
+            <ul>
+              <li class="form-field">
+                <input placeholder="Full name" class="input" type="text" minlength="1" maxlength="30" name="name"
+                  required>
+              </li>
+              <li class="form-field">
+                <input placeholder="Email" class="input" type="email" name="email" required>
+              </li>
+              <li class="form-field">
+                <textarea placeholder="Write your message here" class="text-area" name="message" required rows="10"
+                  maxlength="500"></textarea>
+              </li>
+              <li>
+                <button class="button" type="submit">Get In Touch</button>
```

Comment:
- [Optional] You have done a fantastic job in this project :clap: What do you think of placing this button at the center?. In the Original design, the button is at the center.